packages/app/client/src/ui/pages/ForgotPassword.js

```diff
@@ -0,0 +1,81 @@
+import React, { useState } from 'react'
+import Container from '@material-ui/core/Container'
+import Grid from '@material-ui/core/Grid'
+import { SimpleFormWithHeader } from '../organisms'
+import { Button, Title } from '../atoms'
+import { password, email } from '../../utils/formFields'
+import { useContainerStyle } from '../hooks'
+import { validateEmail, validatePassword } from '../../utils/validators'
+import { sendForgotPasswordEmail, changeUserPassword } from '../../api/users'
+
+const formProps = [
+  {
+    title: "Forgot password",
+    titleProps: {
+      alignMobile: "left",
+      align: "center"
+    },
+    validate: ({ email }) => validateEmail(email),
+    submitLabel: 'Reset password',
+    fields: [email],
+    buttonLayout: { xs: 8, sm: 8 }
+  },
+  {
+    title: "Reset password",
+    titleProps: {
+      alignMobile: "left",
+      align: "center"
+    },
+    validate: ({ password }) => validatePassword(password),
+    submitLabel: 'Confirm password',
+    fields: [password],
+    buttonLayout: { xs: 8, sm: 8 }
+  }
+]
+
+const ForgotPassword = ({ history }) => {
+  const [step, changeStep] = useState(2)
+  const containerStyle = useContainerStyle({ size: 'small' })
+
+  const verifyEmail = async ({ email }) => {
+    try {
+      await sendForgotPasswordEmail({ email })
+      changeStep(1)
+    } catch(e) {
+      // TODO: add handler
+      console.log(e)
```

Comment:
This should be implemented now no?

Reply:
I implemented something to deal with it in this PR -> https://github.com/unicef/atrium/pull/6

Reply:
I don’t If you meant a handler like get the error code and throw a message or show something to the user

Reply:
I am on this. I will change that from the code perspective to handle the error and show it to the user in a better way